src/curl.c

```diff
@@ -480,6 +480,8 @@ CURLcode swupd_curl_set_basic_options(CURL *curl, const char *url)
 		goto exit;
 	}
 
+	curl_easy_setopt(curl, CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_2_0);
+
```

Comment:
If NONE indeed falls back immediately to 1.1, then that is definitely a poor choice. Do you have any references (code or doc's) which clarify that?